File: src/main/java/xyz/oribuin/lilori/commands/author/CmdTest.java
package xyz.oribuin.lilori.commands.author;

import net.dv8tion.jda.api.entities.Emote;
import xyz.oribuin.lilori.managers.command.Command;
import xyz.oribuin.lilori.managers.command.CommandEvent;

import java.util.Collections;

public class CmdTest extends Command {

    public CmdTest() {
        this.name = "Test";
        this.description = "A test command.";
        this.ownerOnly = true;
        this.aliases = Collections.emptyList();
    }

    @Override
    public void executeCommand(CommandEvent event) {
        event.reply("Command Disabled");
    }
}

File: src/main/kotlin/xyz/oribuin/lilori/commands/author/CmdTest.kt
package xyz.oribuin.lilori.commands.author

import net.dv8tion.jda.api.EmbedBuilder
import xyz.oribuin.lilori.LilOri
import xyz.oribuin.lilori.handler.Category
import xyz.oribuin.lilori.handler.Command
import xyz.oribuin.lilori.handler.CommandEvent
import xyz.oribuin.lilori.managers.TicketManager

class CmdTest(bot: LilOri) : Command(bot) {
    init {
        name = "Test"
        category = Category(Category.Type.AUTHOR)
        aliases = emptyList()
        description = "A test command."
        arguments = emptyList()
        isOwnerOnly = true
    }

    override fun executeCommand(event: CommandEvent) {
        event.deleteCmd()

        val embed = EmbedBuilder()
                .setAuthor("Current Ticket Count")
                .setDescription("Ticket Count: ${bot.getManager(TicketManager::class).getTicketCount(event.author)}")
                .setColor(event.color).build()

        event.reply(embed)
    }
}
